code/src/pages/TopRatedMoviesList.js

```diff
@@ -0,0 +1,26 @@
+import React, { useEffect, useState } from 'react';
```

Comment:
Same comment as above. You have done this functionality for the first list of movies. If you want, you could work with props to reuse your files instead of creating new ones for each list 😊